Reference to old boringssl commit causes Eventuals build to fail

I was trying to compile the **3rdparty/eventuals** repo and ran into an issue building the boringssl dependency. Specifically, the build produces the following error:

```
external/boringssl/src/crypto/x509/t_x509.c:321:18: error: variable 'l' set but not used [-Werror,-Wunused-but-set-variable]
    int ret = 0, l, i;
```

The boringssl project fixed this issue with this [commit](https://boringssl.googlesource.com/boringssl/+/af34f6460f0bf99dc267818f02b2936f60a30de7%5E%21/#F0). Unfortunately, it appears [bazel-rules-curl](https://github.com/3rdparty/bazel-rules-curl)/[bazel](https://github.com/3rdparty/bazel-rules-curl/tree/main/bazel)/deps.bzl references a prior commit that includes this unused variable. Would you please update ```deps.bzl``` to point to a newer commit (>= #af34f64) that includes this fix so that the 3rdparty/eventuals repo will continue to compile successfully. I suspect the **3rdparty/eventuals** project may need to be updated as well to reference any update made here.

Thanks . . .

--- a/bazel/deps.bzl
+++ b/bazel/deps.bzl
@@ -11,9 +11,12 @@ def deps(repo_mapping = {}):
     maybe(
         git_repository,
         name = "boringssl",
-        commit = "b9232f9e27e5668bc0414879dcdedb2a59ea75f2",
+        # BoringSSL doesn't usually cut releases, so we use a commit
+        # from the 'main-with-bazel' branch, that has bazel rules for
+        # it's dependencies.
+        commit = "652d66d1feb8ba612e776e03182fa1c8f716d265",
         remote = "https://boringssl.googlesource.com/boringssl",
-        shallow_since = "1637714942 +0000",
+        shallow_since = "1705953338 +0000",
         repo_mapping = repo_mapping,
     )
 
@@ -21,9 +24,9 @@ def deps(repo_mapping = {}):
         new_git_repository,
         name = "boringssl_with_bazel_rules_curl_build_file",
         build_file = "@com_github_3rdparty_bazel_rules_curl//:BUILD.boringssl.bazel",
-        commit = "b9232f9e27e5668bc0414879dcdedb2a59ea75f2",
+        commit = "652d66d1feb8ba612e776e03182fa1c8f716d265",
         remote = "https://boringssl.googlesource.com/boringssl",
-        shallow_since = "1637714942 +0000",
+        shallow_since = "1705953338 +0000",
         repo_mapping = repo_mapping,
     )
 

